app/views/content/terms_candidate.html.erb

```diff
@@ -7,75 +7,90 @@
     <p class="govuk-body">These terms and conditions set out the standards the Department for Education (DfE) expects when you use this service to: </p>
 
     <ul class="govuk-list govuk-list--bullet">
-      <li>submit an application to initial teacher training courses</li>
-      <li>make contact with other people using this service</li>
+      <li>apply to initial teacher training courses</li>
+      <li>contact other people using this service</li>
       <li>link to this service, or interact with it in any other way</li>
     </ul>
 
     <p class="govuk-body">Applying for teacher training using this service means you agree to these terms. If you do not agree, you must stop using the service.</p>
```

Comment:
Should the first part of this sentence be a heading? 'Apply for teacher training (Apply)' then a new para: Using Apply means you agree to these terms...' Or should the sentence be 'By using Apply for teacher training (Apply) you are agreeing to these terms...'

Reply:
I've tweaked this slightly.